tests/subsys/llext/simple/testcase.yaml

```diff
@@ -36,50 +36,6 @@ tests:
     extra_configs:
       - arch:arm:CONFIG_ARM_MPU=n
       - CONFIG_LLEXT_STORAGE_WRITABLE=y
-  llext.simple.modules_enabled_writable:
-    filter: not CONFIG_MPU and not CONFIG_MMU
-    platform_key:
-      - simulation
-      - arch
-    platform_exclude:
-      - qemu_cortex_a9 # MMU
```

Comment:
Cortex-A9 platform exclusion seems gone in `sample.yaml`.
Is this on purpose?

Reply:
Yes, that exclusion was there before the runtime filter was added, so that platform should be caught by the above filter anyway. I will double-check this before submitting v2 though.